src/main/java/mate/academy/spring/dto/request/UserRequestDto.java

```diff
@@ -1,7 +1,15 @@
 package mate.academy.spring.dto.request;
 
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import mate.academy.spring.validation.Email;
+
 public class UserRequestDto {
+    private static final int MIN_PASSWORD_LENGTH = 4;
+    @Email
```

Comment:
now you don't check for null in your validator

Reply:
done

Reply:
is it?